core/arch/arm/tee/cache.c

```diff
@@ -49,13 +49,15 @@ TEE_Result cache_operation(enum utee_cache_operation op, void *va, size_t len)
 
 	switch (op) {
 	case TEE_CACHEFLUSH:
+#ifdef CFG_PL310
```

Comment:
OK, good. Please make the rename a separate patch, before this one (or after, as you prefer).